src/audio/sound_source.hpp

```diff
@@ -30,6 +30,7 @@ class SoundSource
 
   virtual void play() = 0;
   virtual void stop(bool unload_buffer = true) = 0;
+  virtual void pause() {}
```

Comment:
Maybe this function should be pure like the others?

Reply:
I don't think that there will be any improvement after this change. It will require overriding that method in `DummySoundSource` too.

Reply:
There wouldn't be a functional improvement, but will ensure any future sound sources will override this method. Also allows for consistency.